README.md

```diff
@@ -68,14 +68,20 @@ $ tail -f /var/log/td-agent/td-agent.log
   * execute ps command with some options
   * [default] Linux: `LANG=en_US.UTF-8 && ps -ewwo lstart,user:20,pid,ppid,time,%cpu,%mem,rss,sz,s,comm,cmd`
   * [default] MacOSX: `LANG=en_US.UTF-8 && ps -ewwo lstart,user,pid,ppid,time,%cpu,%mem,rss,vsz,state,comm,command`
+  * [default] Windows: described in the next section, `About Windows`.
 
 * keys (Optional)
   * output record keys of the ps command results
-  * [default] start_time user pid parent_pid cpu_time cpu_percent memory_percent mem_rss mem_size state proc_name command
+  * [default] Linux, MacOSX: `start_time,user,pid,parent_pid,cpu_time,cpu_percent,memory_percent,mem_rss,mem_size,state,proc_name,command`
+    * need to fix `command` to match this.
```

Comment:
This description is a bit confusing.
It seems that it says it always requires to modify `command` even if a user doesn't replace this value.

How about this?
`````
need to modify `command` too if you modify this value.
`````

Reply:
Thank you for the advice.
It will be much easier to understand.
I will fix it that way.

Reply:
I have fixed this in https://github.com/daipom/fluent-plugin-watch-process/pull/2/commits/d3fd6eb2f968e7f1b5433acbb65d7fddf27f2b28.